docs/tutorials/how-to-deploy-a-serverless-function.md

```diff
@@ -0,0 +1,217 @@
+  <div id="step1-install-yarn">
+  ```
+  yarn add global genezio
+  ```
+  </div>
+  </TabItem>
+</Tabs>
+
+### 2. Ensure you have a Node.js Function handler
+
+If you don't have a Node.js function handler, you can create one using the following steps:
+
+<details>
+  <summary>**Create a Hello World Serverless Function handler**</summary>
+
+### 1. Initialize a New Node.js Project
+
+Run the following command to initialize a new Node.js project in an empty directory:
+
+```bash
+npm init -y
+```
+
+### 2. Create a Function handler
+
+<Tabs>
+<TabItem className="tab-item" value="esm" label="esm">
+Create a new file named `app.mjs` and add the following code:
+<div>
+```javascript title="app.mjs"
+export const handler = async (event) => {
+  console.log('Function was called');
+  let name = event.queryStringParameters?.name || 'World';
+  return {
+    statusCode: 200,
+    body: `Hello ${name} from serverless function!`,
+  };
+};
+```
+
+  </div>
+  </TabItem>
+  <TabItem className="tab-item" value="cjs" label="cjs">
+  Create a new file named `app.js` and add the following code:
+  <div>
+  ```javascript title="app.js"
+  exports.handler = async (event) => {
+    console.log("Function was called");
+    let name = event.queryStringParameters?.name || "World";
+    return {
+      statusCode: 200,
+      body: `Hello ${name} from serverless function!`,
+    };
+  };
+    ```
+
+  </div>
+  </TabItem>
+</Tabs>
+
+</details>
+
+## Deployment Guide
+
+## 1. Create the Genezio Configuration File
+
+Now, create a `genezio.yaml` file in the root directory of your project.
+
+This file will contain the configuration needed to deploy your backend using Genezio. Here is an example configuration.
+
+:::info
+
+1. You might need to replace the `entry` field with the name of your main application file.
+2. You might need to replace the `path` field with the path relative at **genezio.yaml** file.
+3. This example configuration works if **genezio.yaml** is in the same directory as your main application file.
+   :::
+
+```yaml title="genezio.yaml"
+# The name of the project.
+name: functions-app
+# The region where the project is deployed. Available regions: us-east-1, eu-central-1
+region: us-east-1
+# The version of the Genezio YAML configuration to parse.
+yamlVersion: 2
+backend:
+  # The root directory of the backend.
+  path: ./
+  # Information about the backend's programming language.
+  language:
+    # The name of the programming language.
+    name: js
+    # The package manager used by the backend.
+    packageManager: npm
+  # Information about the backend's functions.
+  functions:
+    # The name (label) of the function.
+    - name: hello-world-function
+      # The path to the function's code.
+      path: ./
+      # The name of the function handler
+      handler: handler
+      # The entry point for the function.
+      entry: app.mjs
+      # The compatibility of the function handler.
+      type: aws
```

Comment:
I believe you can remove this `type` field. From the following CLI release, `aws` will be the default value.
It's not necessary to explicitly add it.